sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/DcosConstants.java

```diff
@@ -23,6 +23,8 @@
     // in general if the network does not support port-mapping, but the task maps the ports the behavior is
     // as expected.
     public static final Boolean DEFAULT_PORT_MAPPING_POLICY = true;
+    public static final Integer OVERLAY_DYNAMIC_PORT_RANGE_START = 1000;
```

Comment:
Should this be 1025? If the task happens to not be running as root it might have problems listening on a port <= 1024.